appSettings.json

```diff
@@ -3,7 +3,9 @@
     "accountEndpoint": "https://your-account-endpoint.documents.azure.com:443/",
     "authKeyOrResourceTokenCredential": "your-auth-key-or-resource-token",
     "cosmosDatabaseId": "vector-benchmarking",
-    "cosmosContainerId": "",
+    "cosmosContainerId": "your-conatiner-id",

```

Comment:
unrelated but need to add 'stopOperationId' to runbook. I missed that. Can you add it?

Reply:
"streaming": {
        "startOperationId": 1,
        **"stoptOperationId": 1001,**
        "totalVectors": 3000000
      }